DogsList/DogsList/Modules/Weather/Model/WeatherManager.swift

```diff
@@ -101,7 +101,7 @@ struct WeatherManager {
             return forecast
             
         } catch {
-            delegate?.didFailWithError(error: error)
+            //delegate?.didFailWithError(error: error)
```

Comment:
do not include in pr commented out code.

Reply:
Ok. Fixed.